components/atoms/BorrowButton.tsx

```diff
@@ -1,8 +1,20 @@
 import React from "react";
 import { useNavigation } from '@react-navigation/native';
 import {Button} from "./Button";
+import { StyleSheet } from "react-native";
 
 export const BorrowButton = () => {
     const navigation = useNavigation();
-    return <Button title={"Borrow"} onPress={() => navigation.navigate("Borrow")} />
+    return <Button title={"Borrow"} onPress={() => navigation.navigate("Borrow")} style={styles.button} />
 };
+
+const styles = StyleSheet.create({
+    button: {
+        borderTopLeftRadius: 0,
+        borderBottomLeftRadius: 0,
+        borderBottomRightRadius: 20,
+        borderTopRightRadius: 20,
```

Comment:
What are these numbers? Can we use somehow `STYLES` here?, something like `STYLES.buttonHeight/2` or something like that